packages/nc-gui/components/account/Token.vue

```diff
@@ -73,6 +73,7 @@ loadTokens()
 const isModalOpen = ref(false)
 const tokenDesc = ref('')
 const tokenToCopy = ref('')
+const isTextEmpty = ref(false)
```

Comment:
This needs a better name. Without checking the full code, I've no idea what this `Text` means.